addons/l10n_latam_invoice_document/wizards/account_debit_note.py

```diff
@@ -0,0 +1,15 @@
+from odoo import models
+
+
+class AccountDebitNote(models.TransientModel):
+
+    _inherit = 'account.debit.note'
+
+    def create_debit(self):
+        """ Properly compute the latam document type of type debit note. """
+        res = super().create_debit()
+        new_move_id = res.get('res_id')
+        if new_move_id:
+            new_move = self.env['account.move'].browse(new_move_id)
+            new_move.with_context(internal_type='debit_note')._compute_l10n_latam_document_type()
```

Comment:
Does this internal_type makes sense?  I mean, now that you know when it is a debit note or not through the link, do we still need this double type in the context?

Reply:
@jco-odoo  thank you for the merge. yes we need the internal_type in order to properly compute the document type that is save by default when creating the debit note, this way the new invoice will one of the debit note document types we have

![image](https://user-images.githubusercontent.com/7593953/90143124-9c264800-dd53-11ea-925e-c3953d4433ab.png)

Best Regards